pkg/framework/machinesets.go

```diff
@@ -37,6 +43,10 @@ const (
 	machineAPIGroup = "machine.openshift.io"
 )
 
+var (
+	ErrMachineCreateFailedInsufficientCapacity = errors.New("machine creation failed due to insufficient capacity")
```

Comment:
Does this need to be exported? Please make sure to add a comment describing what this is

Reply:
Yes, it is used from the spot package. Added comment